server/socket_io/GameIO.js

```diff
@@ -116,9 +114,11 @@ class GameIO {
 
         if (match) {
           match.resetGame();
+          match.startTimerInterval(this.gameIO, matchId);
           this.gameIO
             .to(matchId)
             .emit("game state change", match.getGameState());
+          this.gameIO.to(matchId).emit("start timer");
```

Comment:
I don't think we need this emit anymore now that we listen to gameState.timeEnd's change on the frontend

Reply:
Yeah, I forget to delete it.